src/components/Welcome/Welcome.jsx

```diff
@@ -1,6 +1,12 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Welcome.css';
 
-// Create a Welcome function returning the HTML of welcome block
+const Welcome = () => (
+  <>
+    <section className="welcome">
+      <span className="welcome__text">Sticky Header!</span>
+    </section>
+  </>
```

Comment:
```suggestion
    <section className="welcome">
      <span className="welcome__text">Sticky Header!</span>
    </section>
```

you don't need to use React.Fragment here! You already have one parent element `<section></section>`